CHANGELOG.md

```diff
@@ -5,6 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.0] (2021-02-09)
+
+### Added
+
+- `poll` command
+- `cloudmap` command
```

Comment:
I wouldn't call `cloudmap` a command, since it comes after the actual `poll` command. Maybe *subcommand* or *option*? I know that from the code perspective it's a command, but this is more user facing documentation.

Reply:
I am calling them commands as that is usually what the shell calls them too and other projects as well, so to be consistent. Example:

```bash
% git asd
git: 'asd' is not a git command. See 'git --help'.
```

```bash
$ kubectl config test
error: unknown command "test"
See 'kubectl config -h' for help and examples
```

Reply:
Ok, fair enugh :)